setup.py

```diff
@@ -28,27 +28,31 @@ def find_version(*file_paths):
     description=__doc__,
     long_description=read("README.md"),
     long_description_content_type="text/markdown",
-    author="eduardomourar",
-    author_email="eduardomourar@users.noreply.github.com",
-    url="https://github.com/eduardomourar/cloudformation-cli-typescript-plugin",
+    author="Amazon Web Services",
+    author_email="aws-cloudformation-developers@amazon.com",
+    url="https://github.com/aws-cloudformation/cloudformation-cli-typescript-plugin",
     packages=["rpdk.typescript"],
     package_dir={"": "python"},
     # package_data -> use MANIFEST.in instead
     include_package_data=True,
     zip_safe=True,
     python_requires=">=3.6",
-    install_requires=["cloudformation-cli>=0.1.14", "zipfile38>=0.0.3,<0.2"],
+    install_requires=[
+        "cloudformation-cli>=0.1.14",
+        "docker>=3.7,<5",
```

Comment:
What's the reason for pinning `docker` versions? (since `docker` is already an indirect dependency through `cloudformation-cli`)

Reply:
Legacy code. It was added when cfn-CLI didn't have docker. And because we have a implicit dependency on SAM CLI which does require docker: https://github.com/aws-cloudformation/cloudformation-cli-typescript-plugin/blob/master/python/rpdk/typescript/codegen.py